tests/Feature/ApplicationTest.php

```
@@ -630,6 +630,44 @@ public function test_secret_values_preserved_when_updating_with_empty_value(): v
         $this->assertEquals('original_secret', $storedVar['value']);
     }
 
+    public function test_secret_values_reflect_live_server_file_when_changed_out_of_band(): void
+    {
+        $this->actingAs($this->user);
+
+        // Save a secret through Vito so it is also stored in the DB.
+        SSH::fake('DB_PASSWORD=original_secret');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Facades\SSH' in method 'test_secret_values_reflect_live_server_file_when_changed_out_of_band'. (undefined)

(StaticAccess)